code/datums/brain_damage/imaginary_friend.dm

```diff
@@ -191,6 +196,13 @@
 		return FALSE
 	abstract_move(owner)
 
+/mob/camera/imaginary_friend/pointed(atom/A as mob|obj|turf in view())
+	if(!..())
+		return FALSE
+	to_chat(owner, "<b>[src]</b> points at [A].")
+	to_chat(src, "<span class='notice'>You point at [A].</span>")
+	return TRUE
```

Comment:
Will this display the point arrow and animation?

Reply:
Yes, anything that can see the pointer will see the animation. This is just for the `to_chat`s, which come from `/mob/living/pointed`.

See `/mob/dead/observer/pointed`, which does the same thing.